app/tests/workstations_tests/test_session_control.py

```diff
@@ -0,0 +1,79 @@
+import json
+from urllib.parse import urlparse
+
+import pytest
+from django.conf import settings
+from django.views.generic import TemplateView
+
+
+class SessionControlView(TemplateView):
+    template_name = "session_control.html"
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data()
+        context.update(
+            {
+                "netloc": settings.DJANGO_LIVE_TEST_SERVER_ADDRESS,
+                "domain": settings.DJANGO_LIVE_TEST_SERVER_ADDRESS.split(":")[
+                    0
+                ],
+            }
+        )
+        return context
+
+
+class WorkstationView(TemplateView):
+    template_name = "workstation.html"
+
+
+class SessionCreationView(TemplateView):
+    template_name = "new_session.html"
+
+
+@pytest.mark.playwright
+def test_viewer_session_control(live_server, page, settings):
+    url = urlparse(live_server.url)
+    settings.DJANGO_LIVE_TEST_SERVER_ADDRESS = f"{url.netloc}"
+    session_create_view = f"{url.scheme}://{url.netloc}/new-session/"
+    session_control_view = f"{url.scheme}://{url.netloc}/session-control/"
+    mock_workstation_view = f"{url.scheme}://{url.netloc}/workstation/"
```

Comment:
Why are the URLs hardcoded and not using `reverse` in the tests or `{% url ... %}` in the templates?

Reply:
Makes sense to use {% url %} in the templates. In the tests we still need to construct the urls so that we get absolute urls and not just the paths, but I will replace the hardcoded paths with reverse()